HISTORY.md

```diff
@@ -1,6 +1,8 @@
-# History
+History
```

Comment:
These format changes are not necessary.
Check comment below about using `mdinclude` in `history.rst`